stdlib/2/SimpleHTTPServer.pyi

```diff
@@ -0,0 +1,16 @@
+# Stubs for SimpleHTTPServer (Python 2)
+
+from typing import Any, AnyStr, IO, Mapping, Optional, Union
+import BaseHTTPServer
+from StringIO import StringIO
+
+class SimpleHTTPRequestHandler(BaseHTTPServer.BaseHTTPRequestHandler):
+    server_version = ...  # type: str
+    def do_GET(self) -> None: ...
+    def do_HEAD(self) -> None: ...
+    def send_head(self) -> Optional[IO[str]]: ...
+    def list_directory(self, path: AnyStr) -> Optional[StringIO]: ...
```

Comment:
This argument should be a Union. listdir takes AnyStr because its return value depends on the type passed in, but that's not true for this function.